package.json

```diff
@@ -143,6 +143,7 @@
     "dotenv": "~10.0.0",
     "ejs": "^3.1.7",
     "enhanced-resolve": "^5.8.3",
+    "esbuild": "^0.17.5",
```

Comment:
We previously missed this package and instead relied on `@angular-devkit/build-angular` to install it.